recipe/conda_build_config.yaml

```diff
@@ -1,24 +1,56 @@
-c_stdlib:          # [linux64]
-  - sysroot        # [linux64]
-c_stdlib_version:  # [linux64 or (osx and x86_64)]
-  - 2.17           # [linux64]
+# for linux, see below
+c_stdlib_version:  # [osx and x86_64]
   - 10.15          # [osx and x86_64]
 
 MACOSX_SDK_VERSION:         # [osx]
   - 12.3                    # [osx]
 
 blas_impl:
-  - mkl                        # [x86 or x86_64]
+  - mkl                         # [x86_64]
   - generic
 
-github_actions_labels:
-- cirun-openstack-gpu-large    # [x86 or x86_64]
-- cirun-openstack-cpu-large
-
 megabuild:
 - true      # [linux]
 - false     # [osx]
 
-zip_keys:
-  - - github_actions_labels
-    - blas_impl
+# due to conda-build limitations, we need to override the entire mega-zip
+# just to set the GHA labels correctly; on linux the list is currently
+# [no-CUDA, CUDA 11.8, CUDA 12.0], i.e. need to respect 12.0 migrator.
+#
+# depends on https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5903
+c_compiler_version:         # [linux]
+  - 12                      # [linux]
+  - 11                      # [linux]
+  - 12                      # [linux]
+cxx_compiler_version:       # [linux]
+  - 12                      # [linux]
+  - 11                      # [linux]
+  - 12                      # [linux]
+fortran_compiler_version:   # [linux]
+  - 12                      # [linux]
+  - 11                      # [linux]
+  - 12                      # [linux]
+c_stdlib_version:           # [linux]
+  - 2.17                    # [linux]
+  - 2.17                    # [linux]
+  - 2.17                    # [linux]
+cdt_name:                   # [linux]
+  - cos7                    # [linux]
+  - cos7                    # [linux]
+  - cos7                    # [linux]
+cuda_compiler:              # [linux]
+  - None                    # [linux]
+  - nvcc                    # [linux]
+  - cuda-nvcc               # [linux]
+cuda_compiler_version:      # [linux]
+  - None                    # [linux]
+  - 11.8                    # [linux]
+  - 12.0                    # [linux]
+docker_image:                                   # [linux]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux]
+  - quay.io/condaforge/linux-anvil-cuda:11.8    # [linux]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux]
+github_actions_labels:                          # [linux]
+  - cirun-openstack-cpu-large                   # [linux]
+  - cirun-openstack-gpu-2xlarge                 # [linux]
+  - cirun-openstack-gpu-2xlarge                 # [linux]
```

Comment:
Let's just use `cirun-openstack-cpu-2xlarge` everywhere to keep this file manageable.

Reply:
OK, fine by me.